### DISCUSSION TITLE
Change license to Creative Commons BY-NC-SA

### DISCUSSION THREAD

**Discussion by Dakkaron** [sanni/cartreader#243] [Ideas & Suggestions]
In reference to the current Kickstarter fiasco authored by user @SAVE-THE-HERO, I'd propose to change the license to Creative Commons 4.0 BY-NC-SA (https://creativecommons.org/licenses/by-nc-sa/4.0/).

This would mean:
-) Attribution
-) Non-Commercial
-) Share Alike (share changes under the same license)

We could consider a dual-licensing as CC 4.0 BY-SA, which would allow commercial use, but only after permission on a case-by-case basis by sanni.

What do you think?

**Comment by tamanegitaro** [comment]
I second you Dakkaron. I was involved in this project to make people happy. I think it’s ok to sell cart reader for who don’t have enough skills to make one by themselves and also to get more development resource. But not like this Save the Hero project. Selling one for $100 is making too much profit. It should be much cheaper like $40-$50 so that everyone can buy one.

**Reply by SAVE-THE-HERO** [reply]
NO WAY!!!!!
I didn't want to write anything here.
Please, please, please read this tweet.
https://twitter.com/scrap_a_/status/1434137146625978370?s=20

**Reply by Androxilogin** [reply]
I don't find the price to be an issue. I've spoke with Sanni about this in the past after making my first one, stating that charging a few hundred was unreasonable to which he clarified he felt it was justified. I didn't think so at the time, but he's right about that. 

I put out a small order of 10 and the price really racks up- not to mention the time it takes to put it all together, testing, programming, printing the case and buying the screws, even. $100 US would actually be a low price but looking over this particular project it seems the components are cheap as they come and the case is the more basic needs version without even adding a clock generator which I would imagine to be a standard. Also keeping track of all the components is a job in itself. I can't imagine having a huge order of all these tasks and doing a quality job. 

But yeah, I've sold a few of these myself. I charge $250 without skimping up on the essentials. When you think about it, a simple Gameboy reader is $50. A Super UFO Pro 8 is around $400 these days. The price really teters to the point of that middle ground. I don't go out there promoting that people buy them or anything but if someone happens to stumble on to my page every once in a while and decides to get one having known everything they possibly can including the alternatives, I'll sell them one, sure. 

There are many aspects of it where I find my own personal limitations I won't cross but what it all comes down to is whatever the project contributers recommend as far as regulations, I'll easily abide by.

Mass producing hundreds of them is one problem I would have with the matter in the developer's shoes. Being built with cheap components and little testing with the names of developers unwittingly slapped onto it I can see being another problem. I've built them for prototype collectors and such for preservation and deliberately shared all links so they knew fully well they could build it themselves for cheaper provided they had the tools and skills. I even made an hour long tutorial explaining the process step by step to show people they don't need to buy one from me. 

Flooding the message boards with questions about faulty hardware out of the project hands can be an issue, and with something such as this, I'm sure it will. No one wants to waste time answering the same questions over and over, it staggeringly cripples productivity. But most of all, promising a project to people throwing their money down to make something happen that already exists while basically boasting as though they created everything to do with the project, basically calling it their own thing.

While making the tutorial video I was hoping it wouldn't spark something such as this and held my own reservations in that up until I completed it. I held off releasing it for a little while. Ultimately I felt it could do more good for the community than bad by opening new developers eyes to join and contribute but now at this point I can't help but reawaken the feeling of being at least somewhat responsible for 'spreading the word' in a negative way by sharing it. 

And I mention this all not to say, "him bad, me good". I just want to state in relation to the fairness of the way this bares on typical low-level sales.

**Reply by SAVE-THE-HERO** [reply]
Hi
Androxilogin
I have read your text.
Thank you .
Clock generator is included as standard.
I didn't want to write anything here.

So this is last
If you have something to say to me, please say [here](url). or [just to me](https://twitter.com/suneon1)
I'll get back to you (when I can).

Thank you
Regards

**Reply by Androxilogin** [reply]
I was actually just sharing my views with the poster before you. Your comment just happened to pop up before I could press 'send'.

**Reply by Dakkaron** [reply]
@Androxilogin That's why I'd recommend a case-by-case dual license. In this case, someone like you can ask if they are allowed to, say, build 10 devices or 2 a month or something and get permission for that. Or someone can setup a larger production, after sanni (or someone else) vetted them and checks that they know their stuff.

The biggest problem that I have with @SAVE-THE-HERO is, that they aren't very knowledgeable. They don't even know how to figure out the polarity on a capacitor. And now there might be 100+ new users with faulty devices and no knowledge (or even a soldering iron) on how to fix or even diagnose problems. That means, @SAVE-THE-HERO rakes in the cash, and people in this repo will have to do support for free.

If @SAVE-THE-HERO was someone known, who already build that thing a few times, and knows how the device works, then the story would be different. But right now, if one of their customers stumbles upon e.g. a software bug, all they could do is reference back to us, since there are no programming skills. This kinda sucks.

**Reply by Dakkaron** [reply]
@SAVE-THE-HERO This is not your project. You cannot dictate where people have to write.

**Reply by Androxilogin** [reply]
> @Dakkaron 

I agree completely. It's definitely unfair to have a free support network for users who don't understand how any of the assembly came together to flood in with problems that shouldn't even be. I'm actually surprised this is just now coming to be an issue. Perhaps set up a mandatory donation of a certain amount in some way, even. I always thought there should be a bit more fine print here, myself. It's an extremely valuable resource which should definitely be more regulated.

**Reply by Dakkaron** [reply]
Well, I guess it didn't have to be regulated so far. Regulations usually are put in place once they are necessary.

@SAVE-THE-HERO posted a link to one of their tweets (https://twitter.com/scrap_a_/status/1434137146625978370?s=20) where they said, that their hands are full of burns from soldering... Doesn't sound like great soldering skills.

**Reply by SAVE-THE-HERO** [reply]
I am narrow minded man.
oh,I've written here again...

You told me to put the link in, now you want me to take it out!?

Is it so bad that I heard about polarity?
I listened because I thought it would be better to listen and make the right thing, rather than give people the wrong thing, even if it meant embarrassing them.
But That was certainly an embarrassing question. 
I got a reply from sanni, which made me blush.lol

I'm starting to get a bit of a headache from the conversation.
Because I'm getting confused.

So This is last...!!
If you have something to say to me, please say [here](https://github.com/sanni/cartreader/discussions/241). or [just to me](https://twitter.com/suneon1)
I'll get back to you (when I can).

Thank you
Regards

**Reply by Androxilogin** [reply]
@Dakkaron Oh yeah, that doesn't sound good at all. I'm curious to see some solder joints with that sort of talk. I don't mean to judge but this is definitely a much bigger problem.

@SAVE-THE-HERO The conversation is going on here with or without your approval or involvement. I'd imagine no one will be actively pursuing your invitation to climb in to your comfort zone for an interview. 

Questions are irrelevant at this point. You've presented an issue- basically all there is to do is await the input of others and clarification on the guidelines everyone who made this project what it is agrees to if there happen to be changes on the horizon and abide by them.

**Reply by scrap-a** [reply]
@SAVE-THE-HERO 
Please do not use my tweets for discussion without permission.
And perhaps, you need to pay a little more respect to great projects and members.

Thank you for reading my poor english.

**Reply by Dakkaron** [reply]
@SAVE-THE-HERO If you are a beginner, have no clue about soldering and ask a question, that's all good. Trying to lean is great.

But if you are a beginner, have no clue about soldering and want to sell your practice pieces as professionally made devices, that's really really bad.

I am soldering since about 15 years, as a hobby. I think I can make some decent work. I wouldn't dream about selling my work, because I have seen professionals at soldering, and I am far from that.

Just as a self-check fo you: Do you use a high-quality soldering microscope to verify your solder joints? Do you use an oscilloscope to verify your connections? Do you use ESD equipment to make sure you don't destroy your work pieces while soldering?

**Reply by blackbird-pixel** [reply]
I have been undertaking the production of cartreader on a small scale for some time. So I am interested in this issue, including whether to change the license.

He says "The originator of cartreader sanni is not opposed to this project."

Indeed, sanni has replied to other discussions, but not to this issue at all. It makes me wonder if that means he thinks this issue is not a problem...

**Reply by SAVE-THE-HERO** [reply]
Please allow me to post a just one link here.
https://www.kickstarter.com/projects/save-the-hero/salvage-writes-save-data-to-your-cartridge/comments

**Reply by Androxilogin** [reply]
The whole "he is not opposed" is based on your interaction of stating and him not responding.. You are decieving people with this blatent unfounded, unvarified assumption. Perhaps he's just modest and doesn't want to deal with it.

You also stated a quote from this thread, "Other guy say "you have to sell 40~50$, 99$ is too expensive"" <sic>

Then proceeded to say that dropping the price to that recommended amount of a single opinion here is essentially not an option. This is false. If you believe this to be valuable information enough to repeat it, you could easily change that, you simply choose not to. I would sell them for cheaper if I was less experienced and used the cheapest parts and components as you have. Not to mention getting crowd funded to just toss them together and throw them into a pile. 

Basically, I see the best course of action of concerned users to report it to have it taken down in this situation. These users can even remain annonymous, avoiding the conversations on the topic all together, in that case. 

You've also mentioned previously the clock generator was supposed to be an "addon" rather than ultimately a vital component until I mention it here. Thee are just soo many red flags with this whole situation. The adverse effects on the people who dedicate themselves to this project could be crippling despite how much you claim to respect the devs.

Also, the user you're interacting with in that thread doesn't seem to understand everything that goes into one of these projects. Having one working unit is simple. Quality assurance is a big issue here especially when working on a bulk amount. The Twitter post you talk about burning yourself with a soldering iron all over as @Dakkaron has pointed out is disturbingly concerning and has been left unacknowledged. 
When anyone proves a point you say that you're not writing here again then leave little snippets here and there that don't change a thing about the subject. 

Maybe he is okay with it, maybe he isn't. Fact is, crowd funding should be morally out of the question unless you're donating a large portion of those proceeds to the developers since you're talking about all this respect you have for these people while openly defying every ounce of their arguments here. 

I have to say, the plan of mass producing is a super dick move. That really takes the whole "hobbyist" out of the equation and destroys the fundamental principals of the whole thing. Rather than inspiring additions here, this whole wiki page will simply become free customer service for someone who destroyed the sanctity of this community. And when people start jumping ship, we'll all have you with your personal greed to thank for that.

For myself, it doesn't matter to me if I can't sell them. I produced a small amount of them, sold a few to people with high interest in contributing to the history of gaming while resting assured they got their dumps as intended. I intended to do this temporarily until they're gone. After that point, to strictly know what they're being used for to decide on agreeing to commission or forwarding to someone else. Making these available to people who understand the complexities of the process is also important. Mass producing them/making them commercially available to everyone could disrupt the whole process. Like N64 prototypes- you could sell someone one of these while they  assume it would be the correct tool to dump the files with. In turn, it erases the cartridge ((the preservation of history)) forever because you failed to disclose the warnings by getting to know your customers on a need-to-know basis. 

Not saying I have all the answers, we all have questions. But creating these is about the personal relationship of relative interest, and that's what I find soo intriguing as well as inspiring about it. Rather than just directing people to the discussions page.

**Comment by sanni** [comment]
I like the GPL license. It worked for the SD2SNES and for the OSSC. Many other licenses have too many restrictions that don't allow the project to grow naturally. 

It is my hope that someday you can just buy an open-source cartridge reader on Aliexpress for $40 or something like that. After all it's just an Arduino with a couple of slots attached. What makes it expensive right now is the amount of manual labor it takes to build and fully test the hardware "thanks" to the current prototype-style design.

So a different hardware design is needed that can be manufactured by machines. Maybe one day someone will come along that has a background in electrical engineering or similar and can improve the hardware.

Therefore I think the GPL license does not need to be exchanged for something else.

**Reply by tamanegitaro** [reply]
OK. Lets stay on GPL license. I also hope someday we can buy cartreader for $40. One thing though, Kickstarter prohibits sharing of previously created project so Save the hero project might be violating Kickstarter’s rule. If anybody think so, you can always report the project in kicksterter’s page. Other than that, I agree that anybody can make cartreader and sell them for whatever the price they want.

I guess Save the hero is just extremely clumsy about expressing his respect to the devs. Otherwise, he wouldn’t open “Thank you” thread.
